### PR TITLE
GSoC refactoring for easier management of organizations

### PR DESCRIPTION
**Pull request by jouvin**
- Organization MD file just contains a front-matter section
- Liquid template defined in an included file and generating all the page contents, including its title

### CODE REVIEW

File: _includes/proposal.ext [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{% comment %}
+Liquid template to display GSoC proposals for a given organization. To be included in a Markdown page.
+Organization is defined by 'organization' attribute in front matter of the parent page.
+Organiization title is defined by 'title' attribute in front matter of the parent page. It defines the proposal
```
Comment (hegner) [inline _includes/proposal.ext]: typo in 'Organiization'

File: _includes/proposal.ext [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+grouping by organization and is case insensitive.
+{% endcomment %}
+
+{:.table .table-hover .table-striped}
+
+# Project Proposals by {{ page.title }}
+
+{% for proposal in site.gsocproposals %}
+{% capture u_proposal_org %}{{ proposal.organization | upcase }}{% endcapture %}
+{% capture u_org %}{{ page.organization | upcase }}{% endcapture %}
```
Comment (hegner) [inline _includes/proposal.ext]: turns out we want to have multiple organizations per project. Can you look into that?

File: gsoc/guideline.md [hunk 3]
```diff
@@ -13,9 +15,18 @@ layout: default
    * add _gsocproposals/proposal_YOURPROJECTyourproposal.md (for example proposal_ROOTspark.md)
    * add a front matter as given in this [example](https://raw.githubusercontent.com/HEP-SF/hep-sf.github.io/master/_gsocproposals/proposal_ROOTspark.md)
    * make a pull request
+   
+## Instructions for Adding a New Organization
+
+Projects are attached to an organization (e.g. CERN, Fermilab...). If you are a new organization, you need to create in `gsoc` directory
```
Comment (hegner) [inline gsoc/guideline.md]: create it in...

File: _includes/proposal.ext (hunk 2 above)
Comment (jouvin) [inline _includes/proposal.ext]: Sure, we can make it a list... I'll try to implement it shortly...

Comment (jouvin) [inline _includes/proposal.ext]: Turned out to be very simple! Liquid is great! I demonsrated how to do it with the project ROOTCReduce: don't merge it like this, I'll remove the list, it was just a demonstration.

**Comment by jouvin** [comment]
@hegner I think I adressed your requested changes... You probably need to acknowledge it.

**Pull request merged**